### ISSUE TITLE
dockerhub missing image tags

### ISSUE THREAD

**Issue by zachgersh** [envoyproxy/envoy#3951]
Hey all,

I was just taking a look at the official dockerhub images hoping to see tagged releases (I only noticed release SHA tags). 

I stumbled across this in the docs:

> We also tag the docker images with release versions when we do official releases.

Wonder why we aren't tagging these images? It is a real pain to have to go figure out which SHA was 1.7.0 when I just want that specific release.

**Comment by mattklein123** [comment]
We do tag releases, but the dockerhub UI only shows the most recent tags. This comes up quite a bit and I don't know any way to handle it better. Suggestions appreciated.

**Comment by zachgersh** [comment]
I was guessing this was probably going to be the issue - apologies for duplicate issue. Maybe we should add a permalink to each one of the release image tags in the readme? Or maybe just build only a nightly + the release version and not upload every successful SHA?

Thoughts?

**Comment by mattklein123** [comment]
Yeah, I think we should just perma link in the docs somewhere, agreed. @richarddli any chance of getting help w/ this?

**Comment by terinjokes** [comment]
@mattklein123 I see there's an image tagged "v1.7.0" but not one tagged "v1.7.1".

```
$ docker pull envoyproxy/envoy:v1.7.1             
Error response from daemon: manifest for envoyproxy/envoy:v1.7.1 not found
```

**Comment by mattklein123** [comment]
I'm going to fix this in the context of https://github.com/envoyproxy/envoy/issues/4052
